frontend/src/app/main/ui/features.cljs

```diff
@@ -0,0 +1,62 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) UXBOX Labs SL
+
+(ns app.main.ui.features
+  (:require
+   [app.common.data :as d]
+   [app.common.logging :as log]
+   [app.main.store :as st]
+   [okulary.core :as l]
+   [potok.core :as ptk]
+   [rumext.alpha :as mf]))
+
+(log/set-level! :debug)
+
+(def features-list #{:auto-layout})
+
+(defn toggle-feature
+  [feature]
+  (ptk/reify ::toggle-feature
+    ptk/UpdateEvent
+    (update [_ state]
+      (log/debug :msg "toggle-feature"
+                 :feature (d/name feature)
+                 :result (if (not (contains? (:features state) feature))
```

Comment:
Maybe it is simple without the not, and changing the order of the enabled/disabled